docs/management_api.yml

```diff
@@ -28,83 +28,80 @@ tags:
   - name: ManagementAPI
 
 paths:
-  /configuration:
+  /config/devices/{deviceId}:
     get:
-      operationId: List Parameters
+      operationId: Get key-value pair store
       tags:
         - ManagementAPI
-      summary: Get existing parameters
-      # parameters: {}
+      summary: Query the configuration store; retrieve all key-value pairs
+      parameters:
+        - in: path
+          name: deviceId
+          schema:
+            type: string
+          required: true
+          description: ID of the device to query.
       responses:
         200:
           description: OK
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: '#/components/schemas/Parameter'
+                type: object
+                properties:
+                  id:
+                    type: string
+                  expected:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/Parameter'
+                  actual:
```

Comment:
To make it consistent with the `reported_ts`, I would rename this field `reported`.

Reply:
I removed all attributes, and the GET endpoint returns only the key-value pairs and not the whole device.

Reply:
Looks better now, but you did not update the API docs. This diff makes it:

```
diff --git a/docs/management_api.yml b/docs/management_api.yml
index 125f4c6..1f4b250 100644
--- a/docs/management_api.yml
+++ b/docs/management_api.yml
@@ -28,7 +28,7 @@ tags:
   - name: ManagementAPI

 paths:
-  /config/devices/{deviceId}:
+  /configurations/devices/{deviceId}:
     get:
       operationId: Get key-value pair store
       tags:
@@ -51,10 +51,10 @@ paths:
                 properties:
                   expected:
                     type: object
-                    $ref: '#/components/schemas/Parameter'
+                    $ref: '#/components/schemas/Configuration'
                   actual:
                     type: object
-                    $ref: '#/components/schemas/Parameter'
+                    $ref: '#/components/schemas/Configuration'
         400:
           description: Bad Request.
           content:
@@ -76,7 +76,7 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/Parameter'
+              $ref: '#/components/schemas/Configuration'
       parameters:
         - in: path
           name: deviceId
@@ -112,15 +112,9 @@ components:
         The JWT can be alternatively passed as a cookie named "JWT".

   schemas:
-    Parameter:
+    Configuration:
       type: object
-      properties:
-        key:
-          type: string
-          description: Configuration key.
-        value:
-          type: string
-          description: Configuration value.
+      additionalProperties: true

     Error:
       type: object
```